new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiSample.java

```diff
@@ -89,7 +91,7 @@
    */
   @BeforeAll
   public static void initAll(@Namespaces(5) List<String> namespaces) {
-
+    logger = ThreadSafeLogger.getLogger();
```

Comment:
you can static import ThreadSafeLogger.getLogger then just call getLogger() here like in other It* classes.